docker-compose.yml

```diff
@@ -42,6 +21,12 @@ services:
       - ./database/snapshots:/snapshots
     ports:
       - 5431:5432
+    command: postgres 
+      -c 'maintenance_work_mem=1GB'
+      -c 'max_wal_size=4GB'
+      -c 'shared_buffers=1GB'
+      -c 'work_mem=32MB'
```

Comment:
https://austininnovation.slack.com/archives/CM9MK950S/p1739554586117019

Reply:
my latest attempt at replicating successfully was with these settings:

```
    command: postgres 
      -c 'maintenance_work_mem=1GB'
      -c 'max_wal_size=5GB'
      -c 'shared_buffers=2GB'
      -c 'work_mem=64MB'
```

all this nonsense would be better served by it's own spinoff issue 😖

Reply:
reporting that these settings helped unblock me on restoring from a snapshot too